templates/log.html

```diff
@@ -1,37 +1,91 @@
 <html>
   <head>
     <title>Log: {{ entry.job.name }} at {{ entry.timestamp }}</title>
-    <link rel="stylesheet" type="text/css" href="/static/style.css" />
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="favicon.ico">
+
+    <title>Cronitor</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="http://getbootstrap.com/dist/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+    <link href="http://getbootstrap.com/assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
+
+    <!-- Custom styles for this template -->
+    <link href="http://getbootstrap.com/examples/starter-template/starter-template.css" rel="stylesheet">
+
+    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
+    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+    <script src="http://getbootstrap.com/assets/js/ie-emulation-modes-warning.js"></script>
+
+    <!-- Custom styles for this template -->
+    <link href="/static/style.css" rel="stylesheet">
+
   </head>
 
   <body>
-    <table class="log_metadata">
-      <tr>
-        <th>Command:</th>
-        <td>{{ entry.command }}</td>
-      </tr>
-      <tr>
-        <th>Timestamp:</th>
-        <td>{{ entry.timestamp }}</td>
-      </tr>
-      <tr>
-        <th>Return Code:</th>
-        <td>{{ entry.rc }}</td>
-      </tr>
-    </table>
-
-    <table class="log_metadata">
-      {% set sorted_env = entry.env.items() %}
-      <!-- {{ sorted_env.sort() }} -->
-      {% for k, v in sorted_env %}
-      <tr>
-        <th>{{ k }}</th>
-        <td>{{ v }}</td>
-      </tr>
-      {% end %}
-    </table>
 
-    <div class="log_text">{% for line, is_err in entry.scan_text() %}{% if is_err %}<span class="log_error">{{ line.rstrip() }}</span>{% else %}{{ line.rstrip() }}{% end %}
-{% end %}</div>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="/">Cronitor Job Dashboard</a>
```

Comment:
Suggest adding a link to the job list here as well. (It's probably a little more friendly than just using the Back button.)